src/Hedgehog/Range.fs

```diff
@@ -212,8 +222,9 @@ module Range =
     /// >>> Range.bounds 99 <| Range.linear 0 10
     /// (0, 10)
     ///
-    let inline linear (x : 'a) : ('a -> Range<'a>) =
-      linearFrom x x
+    [<CompiledName("Linear")>]
+    let inline linear (x : 'a) (y : 'a) : Range<'a> =
+      linearFrom x x y
```

Comment:
I think this is a good change, it won't make a difference to the F# usage and the C# usage will be nicer.